Modules/Setting/Resources/views/admin/module-settings.blade.php

```diff
@@ -5,7 +5,7 @@
     {{ trans('setting::settings.title.module name settings', ['module' => ucfirst($currentModule)]) }}
 </h1>
 <ol class="breadcrumb">
-    <li><a href="#"><i class="fa fa-dashboard"></i> {{ trans('core::core.breadcrumb.home') }}</a></li>
+    <li><a href="{{url('/'.config('asgard.core.core.admin-prefix'))}}"><i class="fa fa-dashboard"></i> {{ trans('core::core.breadcrumb.home') }}</a></li>
```

Comment:
Please add a space in between the `{{` and `}}`

Reply:
Also before/after the `.`

Reply:
You can use the named route for this, rather than a config value: `{{ route('dashboard.index') }}`